tests/time_stream/test_aggregation.py

```diff
@@ -492,6 +521,31 @@ def test_missing_criteria_available(self, _, valid, criteria):
         assert_frame_equal(result.df, expected_df, check_dtype=False, check_column_order=False, check_exact=False)
 
 
+class TestMeanSumWithMissingData(unittest.TestCase):
+    """Tests the MeanSum aggregation with time series that has missing data."""
+    def setUp(self):
+        self.input_ts = ts_PT1H_2days_missing
+        self.target_period = P1D
+        self.column = "value"
+        self.timestamps = [datetime(2025, 1, 1), datetime(2025, 1, 2)]
+        self.expected_counts = [24, 24]
+        self.actual_counts = [20, 21]
+        self.values = {"value": [280.8, 853.7]}
```

Comment:
```suggestion
        self.values = {"value": [280.8, 853.71]}
```

If you just change this to 853.71 then no need to round the results below.  The `assert_frame_equal` already has the `check_exact=False` which I think works to 2dp.